atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -98,6 +98,12 @@ public Duration leaderPingResponseWait() {
         return Duration.ofMillis(leaderPingResponseWaitMs());
     }
 
+    @JsonIgnore
+    @Value.Default
+    public Duration leaderAddressCacheTtl() {
```

Comment:
If we could remove this config, it would be nice.

Reply:
I can remove it when I've reintroduced partitioning, which skips this codepath entirely.